### PR TITLE
Update 0-bug-report.md

### PR DESCRIPTION
**Pull request by guineveresaenger**
Knowing the version helps when troubleshooting.

### CODE REVIEW

**Comment by stack72** [comment]
@guineveresaenger I'd love to see if we can move to the following template structure - https://raw.githubusercontent.com/goreleaser/goreleaser/main/.github/ISSUE_TEMPLATE/bug.yaml

This creates an issue flow as follows:

https://github.com/goreleaser/goreleaser/issues/new?&labels=bug%2Ctriage&template=bug.yaml

What do you think here? I brielfly ran this past @susanev and it went ok :)

**Comment by guineveresaenger** [comment]
@stack72 @susanev - PTAL <3

File: .github/ISSUE_TEMPLATE/bug.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    attributes:
+      label: Actual Behavior
+      description: What happened instead?
+    validations:
+      required: true
+  - type: textarea
+    id: versions
+    attributes:
+      label: Versions used
+      description: Please provide the version of Pulumi and all relevant packages used.
```
Comment (Frassle) [inline .github/ISSUE_TEMPLATE/bug.yaml]: Just ask for the output of `pulumi about` from the project directory.

Comment (guineveresaenger) [reply .github/ISSUE_TEMPLATE/bug.yaml]: ooooo nice, much easier!

File: .github/ISSUE_TEMPLATE/bug.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
+name: Bug Report
+description: Report something that's not working correctly
+labels: kind/bug
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report! Please fill the form below.
```
Comment (susanev) [inline .github/ISSUE_TEMPLATE/bug.yaml:8]: id recommend on avoiding the word please in this template
most ux writing standards recommend leading with the action you want people to take, and please can distract from that. if its important for you to keep it, id keep it here, but then drop it in the rest of the questions.

File: .github/ISSUE_TEMPLATE/bug.yaml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+      label: What happened?
+      description: Please provide a general description of the issue, including what you're trying to accomplish.
+    validations:
+      required: true
+  - type: textarea
+    id: reproducible
+    attributes:
+      label: Steps to reproduce
+      description: |
+        Provide a link to a live example or an unambiguous set of steps to reproduce this bug.
```
Comment (susanev) [inline .github/ISSUE_TEMPLATE/bug.yaml]: can we drop `unambiguous` here? or adjust to `clear` instead?

**Review by susanev — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug.yaml (hunk 2 above)
Comment (guineveresaenger) [reply .github/ISSUE_TEMPLATE/bug.yaml:8]: Dropped.

**Review by mikhailshilkov — APPROVED** [review]
Looks great, I didn't know these issue forms were a thing

**Review by stack72 — APPROVED** [review]

**Pull request merged**